plugins/styleguide/assets/javascripts/discourse/components/styleguide-icons.js

```diff
@@ -22,8 +23,11 @@ export default class StyleguideIcons extends Component {
       ids.push(...Object.keys(REPLACEMENTS));
       this.set("iconIds", [...new Set(ids.sort())]);
     } else {
-      // Let's try again a short time later if there are no svgs loaded yet
-      discourseLater(this, this.setIconIds, 1500);
+      // skip trying it later in debug mode because this was causing the CI to timeout
+      if (!DEBUG) {
```

Comment:
DEBUG is true for development mode and test mode. We still want the discourseLater in dev mode, so I think we should go with `!(DEBUG && isTesting())`.